context/app/static/js/components/files/FileBrowserFile/FileBrowserFile.spec.js

```diff
@@ -65,3 +65,44 @@ test('has correct left margin', () => {
 
   expect(screen.getByTestId('file-indented-div')).toHaveStyle(`margin-left: ${expectedMargin}px`);
 });
+
+test('displays QA chip when is_qa_qc is true', () => {
+  const fileObj = {
+    rel_path: 'fakepath',
+    edam_term: 'faketerm',
+    description: 'fakedescription',
+    file: 'fakefile',
+    is_qa_qc: true,
+    size: 1000,
+  };
+
+  const depth = 0;
+
+  render(
+    <FilesProviders>
+      <FileBrowserFile fileObj={fileObj} depth={depth} />
+    </FilesProviders>,
+  );
+
+  expect(screen.getByText('QA')).toBeInTheDocument();
+});
+
+test('does not display QA chip when is_qa_qc is not provided', () => {
+  const fileObj = {
+    rel_path: 'fakepath',
+    edam_term: 'faketerm',
+    description: 'fakedescription',
+    file: 'fakefile',
+    size: 1000,
+  };
+
+  const depth = 0;
+
+  render(
+    <FilesProviders>
+      <FileBrowserFile fileObj={fileObj} depth={depth} />
+    </FilesProviders>,
+  );
+
+  expect(screen.queryByText('QA')).toBeNull();
+});
```

Comment:
Since it's easy, maybe add a test for the case where `is_qa_qc` is false... Not that we expect that to happen, but if it did, we know what behavior we want.

Reply:
Done, thanks.